src/ngraph/op/util/attr_types.hpp

```diff
@@ -187,6 +187,34 @@ namespace ngraph
         const DiscreteTypeInfo& get_type_info() const override { return type_info; }
     };
 
+    namespace op
+    {
+        enum class BroadcastType
+        {
+            NONE,
```

Comment:
Cherry on top of this cake would be at least a small note, that the meanings of all those values are similar to ones in `AutoBroadcastType`.

Reply:
done, location changed, documentation added